### PR TITLE
bio, credman: support non-preview commands

### PR DESCRIPTION
**Pull request by LDVG**

### CODE REVIEW

File: src/bio.c [hunk 1]
```diff
@@ -66,13 +66,16 @@ bio_tx(fido_dev_t *dev, uint8_t subcmd, cbor_item_t **sub_argv, size_t sub_argc,
 	fido_blob_t	*ecdh = NULL;
 	fido_blob_t	 f;
 	fido_blob_t	 hmac;
-	const uint8_t	 cmd = CTAP_CBOR_BIO_ENROLL_PRE;
+	uint8_t		 cmd;
```
Comment (martelletto) [inline src/bio.c:69]: consider moving the inspection of dev to a local subroutine (e.g. get_cmd() or get_cmd_byte()) and keeping the variable const (same for credman.c)

Comment (LDVG) [reply src/bio.c]: thanks!

**Comment by DEX1999** [comment]
You guys work fast. You answered me about lack of plans for support and that you are welcoming patches. And when i was about to make a PR, i realized that you already make one)

**Comment by LDVG** [comment]
> Your changes are almost identical to what we made in our patch, i can confirm that it works with out devices.

Thanks for checking.  Out of curiosity, what authenticators are you working on? :slightly_smiling_face: 

> When can I expect this to be merged?

Given your feedback, this should be possible to merge relatively soon, pending the usual approval process.

**Comment by DEX1999** [comment]
> Out of curiosity, what authenticator are you working on? 🙂

Just simple CTAP/WEBAUTHN authenticator, nothing special yet, probably out products will come up with something interesting in future :smile:

**Pull request merged**